### PR TITLE
Business logic changes

### PR DESCRIPTION
**Pull request by bgodlin**
Please, notice that the backwards compatibility will break, as new version of Subquery implies the `data` transition from object type to an array

### CODE REVIEW

File: src/mappings/rewardsHandler.ts [hunk 1]
```diff
@@ -10,14 +12,18 @@ export async function rewardsHandler(extrinsic: SubstrateExtrinsic): Promise<voi
     if (record.execution.success) {
 
         let details = new Object();
-        let moneyTransferEvent = extrinsic.events.find(e => e.event.method === 'Transferred' && e.event.section === 'currencies')
-        const { event: { data: [assetId,,,amount] } } = moneyTransferEvent;
-
-        details = {
-            assetId: assetId.toString(),
-            amount: formatU128ToBalance(amount.toString())
+        let rewards = new Array();
```
Comment (Nikita-Polyakov) [inline src/mappings/rewardsHandler.ts]:
```
const rewards = extrinsic.events.reduce((buffer, e) => {
   if (e.event.method === 'Transferred' && e.event.section === 'currencies') {
     const { event: { data: [assetId,,,amount] } } = e;
     buffer.push({assetId: assetId.toString(), amount: amount.toString()});
   }
   return buffer;
}, []);
```

File: src/mappings/poolXYKHandler.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export async function handleXYKPools(block: SubstrateBlock): Promise<void> {
     //todo rework to liquidity proxy quote
     let reserves;
     try {
-        reserves = await api.query.poolXyk.reserves.entries(XOR)
+        reserves = await api.query.poolXYK.reserves.entries(XOR)
```
Comment (Nikita-Polyakov) [inline src/mappings/poolXYKHandler.ts:52]: Where is `api` defined? I cannot see the import of `api`  in this file

Comment (bgodlin) [reply src/mappings/poolXYKHandler.ts:52]: sora-subquery/node_modules/@subql/types/dist/global.d.ts

File: src/mappings/rewardsHandler.ts (hunk 1 above)
Comment (bgodlin) [reply src/mappings/rewardsHandler.ts]: Committed changes

**Review by pgolovkin — APPROVED** [review]

**Review by Nikita-Polyakov — APPROVED** [review]

**Pull request merged**